reports/docusign-autopopulation/index.md

```diff
@@ -0,0 +1,134 @@
+5.	For each field you create, set the default value to point to the data you want to merge (Field from another model is the easiest way to do this).
+6.	Get ready to create your request body!
+
+
+### Creating Your Templated Request Body:
+Because DocuSign’s REST API is fairly complex, creating a templated request body with merge variables is the way to go. (View an example below!)
+You can basically follow the format given in the [DocuSign API Explorer](https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=Envelopes&operations=create) but you will only need the following fields: 
+1.	Envelope
+	- status - either sent or created (for a draft)
+	- templateId - referenced above. 
+	- emailSubject - define the subject of the email to be sent with the DocuSign document. 
+2.	Recipient
+	- name
+	- email
+	- roleName - same as defined in the DocuSign template
+	- recipientId - the best way to get this is to do a [get request for recipients](https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=EnvelopeRecipients&operations=list) on the api explorer 
+	- routingOrder (optional) - can also be defined in DocuSign, but it can be helpful to conditionally choose the routing order in Skuid. 
+3.	Recipient tab
+	- tabLabel - Data Label defined in DocuSign
+	- value - the value you want to insert (use merge syntax here!)
+	- font (optional) - choose a font to stylize your text. (Supported Fonts: Arial, Arial, ArialNarrow, Calibri, CourierNew, Garamond, Georgia, Helvetica,   LucidaConsole, Tahoma, TimesNewRoman, Trebuchet, Verdana, MSGothic, MSMincho, Default)
+
+#### Example Request Body:  
+ 
+    {
+        "status": "sent",
+        "emailSubject": "Please DocuSign this Form",
+        "templateId": "<Your template Id>",
+        "templateRoles": [{
+            "name": "{{RecipientName}}",
+            "email":"recipient@gmail.com",
+            "roleName": "Recipeint1",
+            "recipientId": "1",
+            "routingOrder": "1",
+            "tabs": {
+                "textTabs": [{
+                    "tabLabel": "EmployeeName",
+                    "value": "{{employeeName}}",
+                    "Font": "Calibri"},
+                    {
+                    "tabLabel": "EffectiveDate",
+                    "value": "{{Date}}",
+                    "Font": "Calibri"},                        
+                    {
+                    "tabLabel": "Notes",
+                    "value": "{{Notes}}",
+                    "Font": "Calibri"
+                    }]
+            }
+            
+        },
+        {
+            "name": "{{employeeName}}",
+            "email": "{{employeeEmail}}",
+            "roleName": "Employee",
+            "recipientId": "2",
+            "routingOrder": "2"
+        }]
+    }
+
+### See it in action!: 
```

Comment:
Following punctuation with a colon is kind of awkward. And looking at this now, I'd say the colons on all these headers are not necessary. At the very least, remove it here, but I think you could take them all out.
```suggestion
### See it in action!
```